packages/react/src/__tests__/createReactClassIntegration-test.js

```diff
@@ -475,7 +475,13 @@ describe('create-react-class-integration', () => {
 
     expect(() => {
       ReactDOM.render(<Component />, document.createElement('div'));
-    }).toWarnDev('Defines both componentWillReceiveProps');
+    }).toWarnDev(
+      'Unsafe legacy lifecycles will not be called for components using the new getDerivedStateFromProps() API.\n\n' +
+        'Unknown uses getDerivedStateFromProps() but also contains the following legacy lifecycles:\n' +
```

Comment:
Using "Unknown" as a proper noun feels kind of confusing, but I can't think of something better.

Reply:
Agreed. I wonder if there's a way I could improve this by reformatting the message slightly.

Reply:
Maybe "An unknown component uses..."?

Reply:
That makes it seem like React doesn't know which component, when really- it just doesn't know what the name fo the component is.

Maybe "An unnamed component" ?

Reply:
Actually I think just plain "Component" might be okay, and more inline with other places.

Reply:
In what cases does React not know the name of a component? "An unnamed component" sounds better than "Unknown" to me 👍

Reply:
create-react-class case